src/hilfe/onboarding/onboarding.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: layout-wissensbasis
+breadcrumb:
+  title: Onboarding
+---
+
+
+## Onboarding deiner Mitarbeiter in deine Urlaubsverwaltung 
+
+Es gibt drei unterschiedliche Möglichkeiten Mitarbeiter in deine Urlaubsverwaltung aufzunehmen.
+
+### Mitarbeiter einladen
+
+Nachdem deine Urlaubsverwaltung bereitgestellt wurde, beskommst du von uns einen Link zu deinem Kundenportal.
+In diesem Portal hast du die Möglichkeit anhand von E-Mail-Adressen Mitarbeiter einzuladen die Urlaubsverwaltung zu verwenden.
```

Comment:
In diesem Portal -> Im Kundenportal

Reply:
In diesem Portal hast du die Möglichkeit anhand von E-Mail-Adressen Mitarbeiter einzuladen die Urlaubsverwaltung zu verwenden.
>
Im Kundenportal kannst du neue Mitarbeitende per E-Mail-Adresse eine Einladung zu deiner Urlaubsverwaltung versenden. Die Mitarbeitende erhalten darauf eine Einladung per E-Mail und können ausschließlich mit dieser bei deiner Urlaubsverwaltung registrieren.